src/fs.h

```diff
@@ -92,11 +92,35 @@ static inline auto quoted(const std::string& s)
 }
 
 // Allow safe path append operations.
-static inline path operator+(path p1, path p2)
+static inline path operator/(path p1, path p2)
 {
-    p1 += std::move(p2);
+    p1 /= std::move(p2);
     return p1;
 }
+static inline path operator/(path p1, std::filesystem::path p2)
```

Comment:
Why `std::filesystem::path` for second parameter? Isn't it safer to limit the usage of `std::filesystem::path` only within the `fs::path` wrapper?